helm_deploy/hmpps-interventions-service/templates/cronjob-data-extractor-analytics.yaml

```diff
@@ -15,7 +15,7 @@ spec:
             {{ if (and .Values.env_details.contains_live_data (not .data_protection_impact_assessment_signed_off)) }}
             command: ["sh", "-c", "psql -c 'SELECT 1' && mkdir -p 'export/csv' && touch 'export/csv/signed_off_DPIA_required.csv' && transfer_local_to_s3.sh"]
             {{ else }}
-            command: ["sh", "-c", "extract_psql_all_tables_to_csv.sh && transfer_local_to_s3.sh"]
+            command: ["sh", "-c", "python3 /data-engineering-data-extractor/modules/extract_table_names.py && python3 /data-engineering-data-extractor/modules/extract_pg_jsonl_snapshot.py && transfer_local_to_s3.sh"]
```

Comment:
_nit_
Could Python files as `/data-engineering-data-extractor/modules/extract_table_names.py` have an exec flag and a shebang?

As the `/data-engineering-data-extractor/modules` dir is already in the `$PATH`, the above would allow us to call the chain as
```
extract_table_names.py && extract_pg_jsonl_snapshot.py && transfer_local_to_s3.sh
```

Reply:
Agree. This will be done but with the next set of changes :)